ports/ade/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "ade",
-  "version": "0.1.2",
+  "version": "0.1.2.3",
```

@ras0219-msft @BillyONeal @dan-shaw @JavierMatosD @vicroms 

We discussed this, and the team pointed out that we used version strings for `openssl` which does the same thing. Since there's precedence and our policy is to prefer to show what upstream shows as their version, this should use `version-string` instead.

```suggestion
  "version-string": "0.1.2c",
```

---

thanks, revert to my original changes :)